pages/user-details.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <v-layout column justify-center align-center>
```

Comment:
user-details page can be moved under the a specific directory. So that it would be easier to group other user-related pages.